extensions-core/hdfs-storage/src/main/java/io/druid/storage/hdfs/HdfsDataSegmentPusher.java

```diff
@@ -50,17 +50,20 @@
   private final HdfsDataSegmentPusherConfig config;
   private final Configuration hadoopConfig;
   private final ObjectMapper jsonMapper;
+  private final String pathForHadoop ;
```

Comment:
can we call this "storageDirectory" or "fullyQualifiedStorageDirectory" as that is more commonly understood?